### PR TITLE
Makes legcuffs work once more.

### PR DESCRIPTION
**Pull request by Cameron653**
- Makes legcuffs a child of handcuffs

- Makes legcuffs actually be able to put on people's legs again to force them to walk.

- Fixes the proc that lets you escape legcuffs. It was broken because the if(!canClick()) check or something. I dunno. All I know is that it works now.

Will post screenshots momentarily.
https://i.imgur.com/VxHaFQ4.png (Applying legcuffs)
https://i.imgur.com/4bixFkS.png (What it looks like w/ legcuffs on.)
https://i.imgur.com/69ny4ty.png (If you try to run with legcuffs on)
https://i.imgur.com/KQJOoeR.png (Attempting to and removing legcuffs)
https://i.imgur.com/BfD17hE.png (Sprite updates, legcuffs fall onto floor.)
https://i.imgur.com/D2XC8bS.png (You need to have an aggressive grab to legcuff others!)
https://i.imgur.com/R9cQcI6.png (Sprite works.)

### CODE REVIEW

File: code/game/objects/items/weapons/handcuffs.dm [hunk 1]
```diff
@@ -181,3 +181,84 @@ var/last_chew = 0
...
+	var/mob/living/carbon/human/H = target
+	if(!istype(H))
+		return 0
+
+	if (!H.has_organ_for_slot(slot_legcuffed))
+		user << "<span class='danger'>\The [H] needs at least two ankles before you can cuff them together!</span>"
+		return 0
+
+	if(istype(H.shoes,/obj/item/clothing/shoes/magboots/rig) && !elastic) // Can't cuff someone who's in a deployed hardsuit.
+		user << "<span class='danger'>\The [src] won't fit around \the [H.gloves]!</span>"
```
Comment (Cyantime) [inline code/game/objects/items/weapons/handcuffs.dm]: "H.gloves"

Comment (Cameron653) [reply code/game/objects/items/weapons/handcuffs.dm]: Guh, let me fix.

**Comment by Anewbe** [comment]
A different sprite would probably be a good thing... Looking the same as handcuffs is gonna get people confused.

**Comment by Cameron653** [comment]
Eh. I would give it a new sprite, but I'm not a spriter whatsoever, and a quick examine will tell the person looking at it what it really is. Shouldn't be _too_ big of an issue, seeing as legcuffs aren't too common of a thing.

**Comment by SpadesNeil** [comment]
Let me bug some of our sprite artists and see if we can get something neato.

**Comment by Anewbe** [comment]
@SpadesNeil Anything?

**Comment by Belsima** [comment]
how do you even make legcuffs look different? if you could give me an idea i could try my hand at it

**Comment by SpadesNeil** [comment]
Someone said there's legcuffs already. I suspect it's in the shoes icon files.

**Comment by Anewbe** [comment]
It's literally just a handcuff sprite, from what I can see.

**Comment by Belsima** [comment]
I believe that's referring to the shoes rather than the object you apply to the shoes.

**Comment by SpadesNeil** [comment]
Oh. Well I don't know, then.

**Comment by Cameron653** [comment]
Looking at google images for leg cuff references to post here, but.... Literally every single image looks extremely similar to handcuffs. 
https://i.imgur.com/GBn0O6x.png (Legcuffs)
https://i.imgur.com/fpuuEAf.png   (Handcuffs)
I'm not really how someone could make a sprite that looks too different than handcuffs and still looks like legcuffs. If someone does, feel free to send me the sprites. Other than that, this is ready for merge.

**Comment by Belsima** [comment]
Maybe just for the sake of clarity shift the hue on legcuffs to be a bit darker?

**Comment by Cameron653** [comment]
Can someone that knows how to do that, well, do that and upload the .dmi here and I'll put it in the PR?

**Comment by Cameron653** [comment]
Anything? :v

**Comment by Anewbe** [comment]
I'll go back and make a new sprite later, since it only shows up in like one place.

**Pull request merged**